grumphp.yml.dist

```
@@ -13,9 +13,9 @@ parameters:
 
   extra_tasks:
     phpparser:
-      ignore_patterns: %tasks.phpcs.ignore_patterns%
+      ignore_patterns: [ tasks.phpcs.ignore_patterns% ]
       visitors:
         declare_strict_types: ~
-      triggered_by: %tasks.phpcs.triggered_by%
+      triggered_by: [ tasks.phpcs.triggered_by% ]
```

Is it should be like that because in other components adapted to D9 I see other syntaxis?
```suggestion
      triggered_by: "%tasks.phpcs.triggered_by%"
```

---

Not yet addressed as I can see.
Please see oe_media D9 version how it is done: `ignore_patterns: "%tasks.phpcs.ignore_patterns%"`